roomee-api/roomee-api/Models/Announcement.cs

```diff
@@ -0,0 +1,73 @@
+﻿/* 
+ * Author(s): Majors, Andrew, admajors00@gmail.com
+ * Date Created: March 01 2021
+ * Notes: N/A
+ */
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace roomee_api.Models {
+	public class Announcement {
+		[JsonProperty(PropertyName = "annoucementId")]
+		public int AnnoucementId { get; }
+
+		[JsonProperty(PropertyName = "roomId")]
+		public int RoomId { get; set; }
+
+		[JsonProperty(PropertyName = "createdByUserId")]
```

Comment:
CreatedByUserId and CreationTimestamp won't need setters because the post request from the front-end will not be modifying these. They can be collected from the token with the post request.